packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -163,6 +163,13 @@ export interface PluginConfigTypeAndroid {
    * @see [Android documentation](https://developer.android.com/develop/ui/views/theming/darktheme)
    */
   useDayNightTheme?: boolean;
+
+  /**
+   * Enable JavaScript Bundle compression. Turning this on will result in a smaller APK but may cause significantly slower app startup times.
+   *
+   * @default false
+   */
```

Comment:
```suggestion
  /**
   * Enable JavaScript Bundle compression. Turning this on will result in a smaller APK size but may have slower app startup times.
   *
   * @see [Faster App Startup](https://reactnative.dev/blog/2025/04/08/react-native-0.79#android-faster-app-startup)
   *
   * @default false
   */
```

- add reference link
- remove "significantly" which may not fully correct